src/scheduler/fairshare.c

```diff
@@ -627,7 +627,7 @@ write_usage(char *filename, fairshare_head *fhead)
 	 * ...
 	 */
 
-	strcpy(head.tag, USAGE_MAGIC);
+	pbs_strncpy(head.tag, USAGE_MAGIC, 9);
```

Comment:
You use a combination of static numbers and macros for the size of pbs_strncpy().  Just use sizeof(dest) everywhere.

Reply:
True, It didn't occur to me :)
I'll change it.